es-app/src/guis/GuiCollectionSystemsOptions.cpp

```diff
@@ -86,21 +86,21 @@ void GuiCollectionSystemsOptions::initializeMenu()
 
 
 	// Add option to select default collection for screensaver
-	defaultScreenSaverCollection = std::make_shared< OptionListComponent<std::string> >(mWindow, "DEFAULT COLLECTION TO ADD SCREENSAVER GAMES TO", false);
-	
+	defaultScreenSaverCollection = std::make_shared< OptionListComponent<std::string> >(mWindow, "ADD/REMOVE GAMES WHILE SCREENSAVER TO", false);
 	// Add default option
-	defaultScreenSaverCollection->add("<DEFAULT>", "", Settings::getInstance()->getString("DefaultScreenSaverCollection") == "");
+	std::string defaultScreenSaverCollectionName = Settings::getInstance()->getString("DefaultScreenSaverCollection");
+	defaultScreenSaverCollection->add("Favorites", "", defaultScreenSaverCollectionName == "");
```

Comment:
This is inaccurate - if you are editing a collection and the screensaver starts, pressing the button will add it to the collection that's being edited. That's the default behavior - not favorites, just "the current collection that's being edited". Hence the slightly overly convoluted text here, unfortunately.
Happy to have further suggestions here, but "Favorites" is incorrect in this case.

Reply:
Ah, ok. Got it, when in Collection Editing, that collection takes precedence if <default> is set. If not in Collection Editing <default> relates to the built-in Favorites collection. Will revert that.